devlab/Vagrantfile

```diff
@@ -63,6 +69,12 @@ Vagrant.configure(2) do |config|
             thisnode.vm.provision "shell", path: "./provision/cleanup_nova_instances.sh"
           end
           thisnode.vm.provision "shell", path: "./provision/libvirt.sh"
+        when "lab"
+          thisnode.vm.provision "shell",
+            path: "./provision/cloudferry.sh"
+          thisnode.vm.provision "file",
+            source: "~/.ssh/id_rsa",
+            destination: "/home/vagrant/.ssh/id_rsa" unless not File.exist?("~/.ssh/id_rsa")
```

Comment:
Maybe `if File.exist?()`? Double negation is very difficult to read (unless + not). Also, what happens if `~/.ssh/id_rsa` does not exist?